docs/components/MessagingBubbleView.jsx

```diff
@@ -29,6 +37,42 @@ export default class MessagingBubbleView extends React.PureComponent {
   render() {
     const { theme } = this.state;
 
+    const attachmentsArray = [
+      {
+        key: "1",
+        attachmentID: "1",
+        icon: <FileAttachmentIcon fileType={"doc"} />,
+        onClickAttachment: () => console.log("clicked!"),
+        title: "MyCoolDoclajsdjasldjaslkdjasldkjasldjaslkdjasldjasldjalskjdalskjdaslkjasljd.doc",
+        subtitle: "Click to download",
+      },
+      {
+        key: "2",
+        attachmentID: "2",
+        icon: <FileAttachmentIcon fileType={"image"} />,
+        onClickAttachment: () => console.log("clicked!"),
+        title: "Flyer.png",
+        subtitle: "Click to view",
+      },
+      {
+        key: "3",
+        attachmentID: "3",
+        icon: <FileAttachmentIcon fileType={"audio"} />,
+        onClickAttachment: () => console.log("clicked!"),
+        title: "Morning message.m4a",
+        subtitle: "Click to download",
+      },
+    ].map((attachment) => (
+      <MessagingAttachment
+        key={attachment.key}
+        attachmentID={attachment.attachmentID}
+        icon={attachment.icon}
+        onClickAttachment={attachment.onClickAttachment}
+        title={attachment.title}
+        subtitle={attachment.subtitle}
+      />
+    ));
```

Comment:
dummy attachments to demo what multiple attachments looks like (even though we're currently only implementing one at a time)